### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#2541] [Announcements]
New release, primarily focused around three major new features: full LCM support, completely new Model Merge functionality and Stable-fast compile support  
Also included are several other improvements and large number of hotfixes - see full changelog for details  

Details are in the standard announcement thread:
<https://github.com/vladmandic/automatic/discussions/99#discussioncomment-7653209>

**Comment by afbagwell** [comment]
Maybe I'm missing it, but where did the functionality go in merge that preserves the full model merge for SD 1.x? Right now everything I try is coming out as a pruned model (2 GB).
